src/tools/tool-options.h

```diff
@@ -155,7 +155,7 @@ struct ToolOptions : public Options {
            std::string("Enable ") + description,
            ToolOptionsCategory,
            Arguments::Zero,
-           [=, this](Options*, const std::string&) {
```

Comment:
C++17 only supports `[=]` while C++20 throws a warning that `[=, this]` should be used for clarity since the this-object could also be captured by-value. As there is only one capture + `this` in this lambda anyway, I switched it to an explicit list that won't warn in either mode